portfolio/src/main/java/com/google/sps/data/AUX.java

```diff
@@ -35,22 +36,57 @@ private static void GetDevKey(){
         }
     }
 
-    public static Video VideoIdToObject(String videoId) throws IOException, GeneralSecurityException{
+    public static Video VideoIdToObject(String videoId){
```

Comment:
You could make this throw  VideoException instead of making it a field in the Video class. Then you could catch VideoException in the main method.

Reply:
+1. We can leave it to the caller to handle VideoExceptions.

Reply:
Alright, will do.